workloads/kube-burner/run_nodedensity-cni_test_fromgit.sh

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/bash
+
+WORKLOAD_TEMPLATE=workloads/node-density-cni/node-density-cni.yml
+METRICS_PROFILE=${METRICS_PROFILE:-metrics-profiles/metrics.yaml}
+NODE_COUNT=${NODE_COUNT:-4}
+PODS_PER_NODE=${PODS_PER_NODE:-250}
+export WORKLOAD=node-density-cni
+
+. common.sh
+
+deploy_operator
+check_running_benchmarks
+label_nodes heavy
```

Comment:
why are we labeling nodes w/ "heavy"? same for the other test.

Reply:
Yeh we need *some* label, I just changed it to cni